src/budy/controllers/api/account.py

```
@@ -73,12 +73,25 @@ def update_me(self):
     def avatar_me(self):
         account = budy.BudyAccount.from_session(rules = False)
         avatar = account.avatar
+        if not avatar: return

```

this should raise an exception, remove the return